docs/userguides/gettingstarted.md

```diff
@@ -123,6 +123,41 @@ To learn more about authenticating in the CLI, follow the [Configure profile gui
 
 ## Troubleshooting and support
 
+### PATH issues
```

Comment:
@annie-payseur can I have you look over these new troubleshooting docs?

Reply:
For troubleshooting info, describe the symptom the user experiences, not the cause. So I'd recommend changing this sub-head from "PATH issues" to "Code42 command not found".